aptos-move/move-examples/resource_account/sources/simple_defi.move

```diff
@@ -0,0 +1,135 @@
+/// This module demonstrates how to create a new coin and build simple defi swap functions for the new coin
+/// using a resource account.
+///
+/// - Initialization of this module
+/// Let's say we have an original account at address `0xcafe`. We can use it to call
+/// `create_resource_account_and_publish_package(origin, vector::empty<>(), ...)` - this will create a resource address at
+/// `0b6beee9bc1ad3177403a04efeefb1901c12b7b575ac5124c0205efc0dd2e32a`. The module `simple_defi` will be published under the
+/// resource account's address.
+///
+/// - The basic flow
+/// (1) call create_resource_account_and_publish_package() to publish this module under the resource account's address.
+/// init_module() will be called with resource account's signer as part of publishing the package.
+/// In init_module(),
+/// we create the new coin, and store the signer capability of this resource account, mint capability and burn capability
+/// of the new coin within `ModuleData`.
+/// (2) when exchanging coins, we can call `exchange_to` to exchange `AptosCoin` to `ChloesCoin`, and `exchange_from` to
+/// exchange to `AptosCoin` from `ChloesCoin` in 1:1 ratio
+module resource_account::simple_defi {
+    use std::signer;
+    use std::string;
+
+    use aptos_framework::account;
+    use aptos_framework::coin::{Self, Coin, MintCapability, BurnCapability};
+    use aptos_framework::resource_account;
+    use aptos_framework::aptos_coin::{AptosCoin};
+
+    struct ModuleData has key {
+        resource_signer_cap: account::SignerCapability,
+        burn_cap: BurnCapability<ChloesCoin>,
+        mint_cap: MintCapability<ChloesCoin>,
+    }
+
+    struct ChloesCoin {
```

Comment:
nit: Don't put your name here :) Just use some generic name like ResourceAccountCoin

Reply:
i was using a generic name and david suggested me to put my name here lol

Reply:
Lol I see. It's a nit anyway - I don't feel strongly about it. Up to you :)